LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/CertificateCredentialTest.java

```diff
@@ -214,6 +215,7 @@ public byte[] getExtensionValue(String oid) {
         Assert.assertNotNull(certificateCredential.getPublicCertificate());
     }
 
+    @Ignore
```

Comment:
I believe this test is independent of the changes you are making and test likely still valid

Reply:
reverted